src/main/java/org/maas/Start.java

```diff
@@ -85,11 +82,11 @@ public static void main(String[] args) {
             Initializer init = new VisualizationInitializer();
             sb.append(init.initialize(scenarioDirectory));
         }
-        if(runTimeKeeper) {
```

Comment:
@DharminB with this change you disabled the "-noTK" argument. Was this intentional?

Reply:
Okay, I took a look at why this is the case. It is because the visualization code was merged after the start merges. This kept the old version of the start as the final one. I will submit a merge request to fix this problem right away. Thank you for informing.

Reply:
Would you review #116 please?